rendermodules/mesh_lens_correction/do_mesh_lens_correction.py

```diff
@@ -254,27 +233,49 @@ def run(self):
                     args=['--output_json', out_file.name])
             pmgen.run()
 
+        # load these up in memory to pass to actual solver
+        rawtilespecs = renderapi.tilespec.get_tile_specs_from_z(
+                ts_example['output_stack'],
+                ts_example['z'],
+                render=renderapi.connect(**ts_example['render']))
+        rawtdict = [t.to_dict() for t in rawtilespecs]
+        matches = renderapi.pointmatch.get_matches_within_group(
+                self.args['match_collection'],
+                self.args['sectionId'],
+                render=renderapi.connect(**self.args['render']))
+
         self.logger.setLevel(self.args['log_level'])
 
+        solver_args = {
+                'nvertex': self.args['nvertex'],
+                'regularization': self.args['regularization'],
+                'good_solve': self.args['good_solve'],
+                'tilespecs': rawtdict,
+                'matches': matches,
+                'output_dir': self.args['output_dir'],
+                'outfile': 'resolvedtiles.json.gz',
+                'compress_output': False,
+                'log_level': self.args['log_level']}
+
         meshclass = MeshAndSolveTransform(
-                input_data=args_for_input,
-                args=['--output_json', out_file.name])
+                input_data=solver_args, args=[])
         # find the lens correction, write out to new stack
         meshclass.run()
+        with open(meshclass.args['output_json'], 'r') as f:
+            jout = json.load(f)
+        resolved = renderapi.resolvedtiles.ResolvedTiles(
+                json=jsongz.load(jout['resolved_tiles']))
 
-        if self.args['do_montage_QC']:
-            # run montage qc on the new stack
-            qc_example = self.get_qc_example()
-            qc_mod = DetectMontageDefectsModule(
-                    input_data=qc_example,
-                    args=['--output_json', out_file.name])
-            qc_mod.run()
+        tform_out = os.path.join(
+                self.args['output_dir'],
+                'lens_correction_out.json')
+        with open(tform_out, 'w') as f:
+            json.dump(resolved.transforms[0].to_dict(), f)
 
         try:
             self.output(
-                    {'output_json': meshclass.args['outfile'],
-                     'maskUrl': self.maskUrl,
-                     'qc_json': out_file.name})
+                    {'output_json': tform_out,
+                     'maskUrl': self.maskUrl})
```

Comment:
Are we abandoning lens correction QC altogether?   Was there any logic (computer or human) that previously depended on this?

Reply:
We had turned it off at some point because something in QC was hanging (some sort of tree was getting too large with the number of point matches in lens generation.) We got along fine without it. Initially, I had that because I thought this was effectively a montage, it should pass montage QC, but, we have other checks in place that seem better suited to catching problems with lens correction. So, yes, I propose we abandon it altogether.
I don't think anything depended on this being in the output json. I certainly never looked at it.

Reply:
Cool, let's drop it.